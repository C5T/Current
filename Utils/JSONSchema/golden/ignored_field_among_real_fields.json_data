{
  "a": "foo",
  "b": null,
  "c": "bar"
}
