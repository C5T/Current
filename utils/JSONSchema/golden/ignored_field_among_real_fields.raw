{"Object":{"field_schema":[["a",{"String":{"values":{"foo":1},"counters":[[1,"foo"]],"instances":1,"nulls":0}}],["b",{"Null":{"occurrences":1}}],["c",{"String":{"values":{"bar":1},"counters":[[1,"bar"]],"instances":1,"nulls":0}}]],"field_index":{"b":1,"c":2,"a":0},"instances":1,"nulls":0}}