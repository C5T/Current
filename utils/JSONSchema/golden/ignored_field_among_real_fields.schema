// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(a, std::string);
  // `b` : `null`-s and/or empty arrays, ignored in the schema.
  CURRENT_FIELD(c, std::string);
};

using Schema = Schema_Object;
