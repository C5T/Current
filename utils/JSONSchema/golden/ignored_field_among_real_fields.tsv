Field	Type	Set	Unset/Null	Values	Details
Schema	Object	1	0	3 fields	a, b, c
Schema.a	String	1	0	1 distinct value, "foo":1
Schema.b	Null	1
Schema.c	String	1	0	1 distinct value, "bar":1
